### DISCUSSION TITLE
Does solaris host even support "sync folder"?

### DISCUSSION THREAD

**Discussion by yogapugazh** [vagrant-libvirt/vagrant-libvirt#1819] [Q&A] [answered]
Hi,

While trying to syncing folders between host and guest in the solaris host, it throws error

**Vagrant attempted to execute the capability 'mount_nfs_folder'
on the detect guest OS 'solaris11', but the guest doesn't
support that capability. This capability is required for your
configuration of Vagrant. Please either reconfigure Vagrant to
avoid this capability or fix the issue by creating the capability.**

Vagrantfile contains 
config.vm.synced_folder "./", "/vagrant", type: "nfs", mount_options: ['actimeo=2'], nfs_version: 4.1

**Comment by abbbi** [comment, marked as answer]
See:

https://github.com/vagrant-libvirt/vagrant-libvirt/issues/1820#issuecomment-2241747758

vagrant has to implement this capability, not vagrant-libvirt.
